pkg/lib/host-connector/types/pcr_manifest.go

```diff
@@ -36,12 +36,6 @@ type Pcr struct {
 	PcrBank SHAAlgorithm `json:"pcr_bank"`
 }
```

Comment:
Are we keeping all "Pcr", "PCR" and "PCRS" structures?

Reply:
Yes, we are keeping all "Pcr", "PCR" and "PCRS" structures for the below reasons:

"Pcr" struct is used to create pcr manifest section in host manifest.
"PCRS" struct is used to create pcrs section in flavor part.
"PCR" struct is used to store PCR index with respective PCR bank.

Reply:
I'm confused by the structure names.  I think this file needs refactoring.  For example, it's using ints instead of PcrIndex and strings instead of SHAAlgorithm.

@isteffyx do you want to do the code clean up after the ESXI/rebase changes?  It's going to need to be done before we merge into the ISecL trunk.